### PR TITLE
bumping packages

### PR DESCRIPTION
**Pull request by vix633**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -26127,7 +26125,7 @@
       "dev": true
     },
     "siren-sdk": {
-      "version": "github:BrightspaceHypermediaComponents/siren-sdk#5d0e59242e2c675263ae962105ca1a8d9a20f17e",
+      "version": "github:BrightspaceHypermediaComponents/siren-sdk#2f32da2a23b05acb6a237975561c531532b264c4",
```
Comment (svanherk) [inline package-lock.json:26128]: FYI @emil-furniss

File: package-lock.json [hunk 2]
```diff
@@ -4860,7 +4858,7 @@
       }
     },
     "d2l-my-courses": {
-      "version": "github:Brightspace/d2l-my-courses-ui#4799a815cb26ec64e63c7aa9cb895fe85fbc24c4",
+      "version": "github:Brightspace/d2l-my-courses-ui#3e0214cff7110126c946c3b246feb3f08718fd60",
```
Comment (svanherk) [inline package-lock.json:4861]: FYI @klhuff

File: package-lock.json [hunk 3]
```diff
@@ -4735,7 +4733,7 @@
       }
     },
     "d2l-html-editor": {
-      "version": "github:Brightspace/d2l-html-editor#0b8f56e00c55198ab7b06ffe79fa4dc3c7578233",
+      "version": "github:Brightspace/d2l-html-editor#de4a31e0b8f55ee4e8b8eee86139d6929ca8b1d3",
```
Comment (svanherk) [inline package-lock.json:4736]: FYI @ianlaidlaw @pbrown-d2l

File: package-lock.json [hunk 4]
```diff
@@ -4559,7 +4559,7 @@
       }
     },
     "d2l-cpd": {
-      "version": "github:Brightspace/continuous-professional-development#3484bce78c8ae72b84ae3fc11aad2d84a7e7c668",
+      "version": "github:Brightspace/continuous-professional-development#f1b4fc45ce95a99480af204fc9daee821b9a0b13",
```
Comment (svanherk) [inline package-lock.json:4562]: FYI @BraydonKains

File: package-lock.json [hunk 5]
```diff
@@ -4492,7 +4492,7 @@
       }
     },
     "d2l-consistent-evaluation": {
-      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#8b543deffea0b071458b93fbd609bfa36e233ff8",
+      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#55c17b737b14dc4d9a2accf183c70fbb92ce77cf",
```
Comment (svanherk) [inline package-lock.json:4495]: FYI @dmackenz

File: package-lock.json [hunk 6]
```diff
@@ -4281,7 +4281,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#aadc1e4d0a8b378d2a390930aab43501d0ef4dec",
+      "version": "github:BrightspaceHypermediaComponents/activities#c0d8fcad83709da682b6e89ecdafbcb570536cf0",
```
Comment (svanherk) [inline package-lock.json:4284]: FYI @BhamelinD2L

File: package-lock.json [hunk 7]
```diff
@@ -4343,7 +4343,7 @@
       }
     },
     "d2l-activity-alignments": {
-      "version": "github:Brightspace/d2l-activity-alignments#f21a5d35b152a5c42039e0e8c34be49048ce3832",
+      "version": "github:Brightspace/d2l-activity-alignments#35527141c78061ef6aa75df18b4cbad7d623e549",
```
Comment (svanherk) [inline package-lock.json:4346]: FYI @gfrancu

File: package-lock.json [hunk 8]
```diff
@@ -1045,9 +1045,9 @@
       }
     },
     "@brightspace-ui-labs/grade-result": {
-      "version": "1.0.9",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/grade-result/-/grade-result-1.0.9.tgz",
-      "integrity": "sha512-XTQ08eULdxNjDgLG6aAXqfZuso/+c3080lcS4qCeD2zzL+SYUIR6W8VKSZyLAezGp5hPy4KuDMhCdoI7LzInlQ==",
+      "version": "1.0.13",
```
Comment (svanherk) [inline package-lock.json:1048]: FYI @dmackenz

File: package-lock.json (hunk 2 above)
Comment (klhuff) [reply package-lock.json:4861]: Sweet, we should mark the story as ready for test.

**Pull request merged**